### PR TITLE
releasenotes029

### PR DESCRIPTION
**Pull request by gdementen**
update release notes

### CODE REVIEW

File: doc/source/changes/version_0_29.rst.inc [hunk 1]
```diff
@@ -1,172 +1,107 @@
...
-    >>> uniform(axes='a=a0..a2')
-    a                   a0                  a1                  a2
-       0.33293756929238394  0.5331412592583252  0.6748786766763107
+* deprecated the ``element_equal()`` function in favor of the :py:obj:`LArray.eq()` method (closes :issue:`630`) to be
+  consistent with other future methods for operations between two arrays.
 
-    generate values following a normal distribution (:math:`\mu` = 1 and :math:`\sigma` = 2 in this example)
+* renamed ``nan_equals`` argument of :py:obj:`LArray.equals()` and :py:obj:`LArray.eq()` methods to ``nans_equal``
+  because it is grammatically more correct and is explained more naturally as "whether two nans should be considered
+  equal".
```
Comment (alixdamman) [inline doc/source/changes/version_0_29.rst.inc]: =D

File: doc/source/changes/version_0_29.rst.inc [hunk 2]
```diff
@@ -1,172 +1,107 @@
...
 
-    generate values by randomly choosing between specified values (5, 10 and 15 in this example), potentially with a
-    specified probability for each value (respectively a 30%, 50%, 20% probability of occurring in this example).
+* allowed arrays to have metadata (e.g. title, description, authors, ...).
 
-    >>> choice([5, 10, 15], p=[0.3, 0.5, 0.2], axes='a=a0,a1;b=b0..b2')
-    a\b  b0  b1  b2
-     a0  15  10  10
-     a1  10   5  10
+  Metadata can be added when creating arrays.
```
Comment (alixdamman) [inline doc/source/changes/version_0_29.rst.inc]: s/arrays./arrays:/

File: doc/source/changes/version_0_29.rst.inc [hunk 3]
```diff
@@ -175,73 +110,155 @@ New features
...
 Miscellaneous improvements
 --------------------------
 
-* saving or loading a session from a file now includes `Axis` and `Group` objects in addition to arrays
-  (closes :issue:`578`):
+* completely rewritten the 'Load And Dump Arrays, Sessions, Axes And Groups' section of the tutorial
+  (closes :issue:`645`)
+
+* saving or loading a session from a file now includes :ref:`api-axis` and :ref:`api-group` objects in addition to
+  arrays (closes :issue:`578`).
```
Comment (alixdamman) [inline doc/source/changes/version_0_29.rst.inc]: s/)./):/

File: doc/source/changes/version_0_29.rst.inc [hunk 4]
```diff
@@ -175,73 +110,155 @@ New features
...
 
-  Note: all axes (groups) of a session are stored in the same CSV file/Excel sheet/HDF group
-  named __axes__ (__groups__).
-
-* made some operations on Excel Workbooks a bit faster by telling Excel to avoid updating the screen when the Excel
-  instance is not visible anyway. This affects all workbooks opened via open_excel as well as read_excel and
-  LArray.to_excel when using the default "xlwings" engine.
-
-* added examples for `read_excel` and `read_hdf` functions (closes :issue:`617`).
+  Note that all axes and groups of a session are stored in the same CSV file/Excel sheet/HDF group named
```
Comment (alixdamman) [inline doc/source/changes/version_0_29.rst.inc]: add ..note: ?

File: doc/source/changes/version_0_29.rst.inc [hunk 5]
```diff
@@ -287,83 +304,74 @@ Miscellaneous improvements
...
-* replaced the `element_equal` function by the `eq` method of LArray (closes :issue:`630`).
-
-* renamed `nan_equals` argument of `equals` and `eq` methods of LArray as `nans_equal`.
-
-* made the `from_series` function support series with multiindex (closes :issue:`465`)
-
-* completely rewritten the 'Load And Dump Arrays, Sessions, Axes And Groups' section of the tutorial
-  (closes :issue:`645`)
+* made some operations on Excel Workbooks a bit faster by telling Excel to avoid updating the screen when the Excel
+  instance is not visible anyway. This affects all workbooks opened via open_excel as well as read_excel and
```
Comment (alixdamman) [inline doc/source/changes/version_0_29.rst.inc]: s/open_excel/:py:obj:`open_excel()`/

File: doc/source/changes/version_0_29.rst.inc [hunk 6]
```diff
@@ -287,83 +304,74 @@ Miscellaneous improvements
...
+  LArray.to_excel when using the default "xlwings" engine.
```
Comment (alixdamman) [inline doc/source/changes/version_0_29.rst.inc]: s/LArray.to_excel/:py:obj:`LArray.to_excel()`/

**Comment by gdementen** [comment]
I fixed all remaining sphinx warnings one gets when generating the docs (and most of them fix some formatting issues).

File: doc/source/changes/version_0_29.rst.inc (hunk 3 above)
Comment (gdementen) [reply doc/source/changes/version_0_29.rst.inc]: I prefer not. I don't like a : followed by another title.

**Pull request merged**